keyboards/ergodox_ez/keymaps/lukaus/keymap.c

```diff
@@ -0,0 +1,878 @@
+#include "ergodox_ez.h"
```

Comment:
Please change Line 1 to `#include QMK_KEYBOARD_H`, then delete Lines 2 and 3.

The `QMK_KEYBOARD_H` inclusion includes both `debug.h` and `action_layer.h` automatically.